packages/cubejs-query-orchestrator/src/driver/BaseDriver.ts

```diff
@@ -86,17 +87,14 @@ const DbTypeValueMatcher = {
   text: () => true
 };
 
-export class BaseDriver {
-  /**
-   * Workaround for Type 'BaseDriver' has no construct signatures.
-   *
-   * @param {Object} [options]
-   */
-  constructor(_options = {}) {
+export class BaseDriver implements DriverInterface {
+  private logger: any;
```

Comment:
protected? Interesting to see that `DatabricksDriver` works with private field from `BaseDriver`. 

<img width="726" alt="image" src="https://user-images.githubusercontent.com/572096/189691567-77b378d3-7b5c-4b91-b30c-6195d9935bad.png">

Reply:
Agreed